.github/workflows/build-test.yaml

```diff
@@ -68,40 +72,41 @@ jobs:
       - uses: "actions/checkout@v3"
       - uses: "actions/setup-go@v3"
         with:
-          go-version: "~1.19.2"
+          go-version: "${{ env.GO_VERSION }}"
+          cache: "true"
       - uses: "authzed/actions/go-test@main"
         with:
           tags: "ci,skipintegrationtests"
           timeout: "10m"
+
   integration:
     name: "Integration Tests"
     runs-on: "ubuntu-latest"
     steps:
       - uses: "actions/checkout@v3"
       - uses: "actions/setup-go@v3"
         with:
-          go-version: "~1.19.2"
+          go-version: "${{ env.GO_VERSION }}"
+          cache: "true"
       - uses: "authzed/actions/go-test@main"
         with:
           tags: "ci,docker"
           timeout: "10m"
           working_directory: "internal/services/integrationtesting"
-      - uses: "authzed/actions/go-test@main"
-        with:
-          tags: "ci,docker"
-          timeout: "5m"
-          working_directory: "cmd/spicedb"
+
   datastore:
     name: "Datastore Tests"
     runs-on: "ubuntu-latest"
     strategy:
+      fail-fast: false
```

Comment:
Why not fail fast?

Reply:
because then it cancels other datastore tests which could build independently and be cancelled. Then we would have to rebuild an additional job.